confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java

```diff
@@ -775,6 +777,65 @@ public DataSet showRegion(GetRegionInfoListPlan getRegionsinfoReq) {
     }
   }
 
+  @Override
+  public DataSet showDataNodes(GetRegionInfoListPlan getRegionsinfoReq) {
+    TSStatus status = confirmLeader();
+    DataNodeInfosResp dataNodeInfosResp = new DataNodeInfosResp();
+    if (status.getCode() == TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
+      List<TDataNodesInfo> dataNodesInfoList = nodeManager.getOnlineDataNodesInfoList();
+      RegionInfoListResp regionsInfoDataSet =
+          (RegionInfoListResp) partitionManager.getRegionInfoList(getRegionsinfoReq);
+      Map<Integer, Integer> dataRegionNumMap = new HashMap<>();
+      Map<Integer, Integer> schemaRegionNumMap = new HashMap<>();
```

Comment:
Add typo for these two map, and you can use AtomicInteger as the map value.

Reply:
fixed~